src/map/map.cpp

```diff
@@ -4843,8 +4845,9 @@ void MapServer::finalize(){
 	// remove all objects on maps
 	for (int i = 0; i < map_num; i++) {
 		struct map_data *mapdata = map_getmapdata(i);
-
+#ifdef DEBUG
```

Comment:
Above the loop the loop the same messages as everywhere should be added.
Maybe harmonize the wording as well!
Cant suggest that on Github Web, sorry!